crates/css-syntax/src/syntax.rs

```diff
@@ -116,7 +85,7 @@ pub fn get_at_rule_descriptor_syntax(
 ) -> Syntax {
     let scopes = scope_chain(scope);
     for scope in scopes {
-        if let Some(scoped) = CSS_REF.atrules.get(scope)
+        if let Some(scoped) = css_ref_data().atrules.get(scope)
```

Comment:
Could we call `css_ref_data()` only once, and remember the result? Or is the cost of calling this repeatedly negligible?